packages/datadog-plugin-aws-sdk/test/bedrock.spec.js

```diff
@@ -0,0 +1,224 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const nock = require('nock')
+const { setup } = require('./spec_helpers')
+
+const serviceName = 'bedrock-service-name-test'
+
+const AI21 = 'AI21'
+const AMAZON = 'AMAZON'
+const ANTHROPIC = 'ANTHROPIC'
+const COHERE = 'COHERE'
+const META = 'META'
+const MISTRAL = 'MISTRAL'
+
+describe('Plugin', () => {
+  describe('aws-sdk (bedrock)', function () {
+    setup()
+
+    withVersions('aws-sdk', ['@aws-sdk/smithy-client', 'aws-sdk'], '>=3', (version, moduleName) => {
+      let AWS
+      let bedrockRuntimeClient
+
+      const bedrockRuntimeClientName =
+        moduleName === '@aws-sdk/smithy-client' ? '@aws-sdk/client-bedrock-runtime' : 'aws-sdk'
+      describe('with configuration', () => {
+        before(() => {
+          return agent.load('aws-sdk')
+        })
+
+        before(done => {
+          AWS = require(`../../../versions/${bedrockRuntimeClientName}@${version}`).get()
```

Comment:
```suggestion
          const requireVersion = version === '3.0.0' ? '3.422.0' : '>=3.422.0'
          AWS = require(`../../../versions/${bedrockRuntimeClientName}@${requireVersion}`).get()
```
might be able to try this for CI, although there is probably a better way to do this once the `withVersions` changes land

Reply:
It works ! Nice trick to bypass the limitation on `withVersions`